skimage/exposure/exposure.py

```diff
@@ -428,7 +436,7 @@ def intensity_range(image, range_values='image', clip_negative=False):
     return i_min, i_max
 
 
-def _output_dtype(dtype_or_range):
+def _output_dtype(dtype_or_range, image_dtype):
```

Comment:
Shouldn't we add an `image_dtype` entry under `Parameters`?

Reply:
sure, I will add it